drivers/oled/oled_driver.c

```diff
@@ -108,6 +108,9 @@ bool            oled_active         = false;
 bool            oled_scrolling      = false;
 uint8_t         oled_rotation       = 0;
 uint8_t         oled_rotation_width = 0;
+uint8_t         oled_scroll_speed   = 0;
```

Comment:
Default to the previous hard coded speed.
```suggestion
uint8_t         oled_scroll_speed   = 3;
```

Reply:
Hi @zvecr and thank you so much for checking over this PR. The `oled_scroll_speed` here holds the after remapping into the 1306's somewhat wonky numbering scheme, and 0 was the previous value used in the command.

You are totally correct that if you wanted to set the speed to the previous default via the new `oled_scroll_set_speed` function you would need to provide 3 as the argument, as the default is the fourth fastest speed and this would then be remapped to 0 in the 1306 speed value.  

It is totally strange.  I have no idea why the ssd1306 has this funky scheme for setting the speed (7 is faster than 0... 3 is the slowest????).  I was on the fence about remapping this to a more understandable 0-7 (fastest to slowest) but decided it would be more approachable for most folks... but it does mean the value of the internal variable `oled_scroll_speed` is less obvious.  I've added a comment to make this clearer.

Reply:
Cheers for adding the comment, I see now where i misunderstood!